21.1.5 update (#1)
DevExpress-Examples/out-of-maintenance-wpf-docklayoutmanager-display-baritems-that-switch-between-mdi-documents: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T192029)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

> # WPF Dock Layout Manager - Add a BarItem that Switch Between All Opened MDI Documents 

Reviewer: ```suggestion
# WPF Dock Layout Manager - Add a BarItem that Switches Between All Opened MDI Documents
```

The example adds a [BarItem](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarItem) with a collection of sub items, that allow you to switch between all opened MDI documents in the [DocumentGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentGroup).

To do this, follow the steps below: 

1. Bind the [BarLinkContainerItem.ItemLinksSource](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarLinkContainerItem.ItemLinksSource) property to the [DocumentGroup.Items](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup.Items) collection.

> 1. Place the DataTemplate with the [BarCheckItem](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarCheckItem) control to the [BarLinkContainerItem.ItemTemplate](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarLinkContainerItem.ItemTemplate) property. 

Reviewer: ```suggestion
1. Place the DataTemplate with the [BarCheckItem](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarCheckItem) control into the [BarLinkContainerItem.ItemTemplate](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarLinkContainerItem.ItemTemplate) property.
```

1. Bind the [BarCheckItem.Content](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarItem.Content) property to the [DocumentPanel.Caption](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.BaseLayoutItem.Caption) property to display DocumentPanel's header value in the BarItem's content.

> 1. Bind the [BarCheckItem.IsChecked](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarCheckItem.IsChecked) property to the [DocumentPanel.IsActive](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.BaseLayoutItem.IsActive) property to allow switching between all [DocumentPanels](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentPanel).

Reviewer: Are _users_ allowed to switch between DocumentPanels? If so, can we say:

...`to allow users to switch between [DocumentPanels](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentPanel).`

Author: ```suggestion
1. Bind the [BarCheckItem.IsChecked](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarCheckItem.IsChecked) property to the [DocumentPanel.IsActive](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.BaseLayoutItem.IsActive) property to allow users switch between [DocumentPanels](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentPanel).

```

Reviewer: Please change "to allow users switch between" to "to allow users to switch between"
